### ISSUE TITLE
Archive this repository

### ISSUE THREAD

**Issue by rgaiacs** [carpentries/2017Merger#13]
@tracykteal Can we archive this repository? Will be more productive for people to link to issues here but not add new comments during the new request for comments.

cc @k8hertweck

**Comment by k8hertweck** [comment]
Agreed; it would be great to make sure this repo isn't confused with the new RFCs
